t/md5.t

```diff
@@ -1,11 +1,20 @@
 use strict;
 use warnings;
 
-use Test::More tests => 3;
+use Test::More tests => 4;
+use File::Temp qw(tempfile);
 use Rex::Commands::MD5;
 
 my $test_file = Rex::Helper::File::Spec->catfile( 't', 'md5test.bin' );
 
+my ( $fh, $strangely_named_file ) = tempfile(
+  "XXXXX",
+  SUFFIX => " = 11111111111111111111111111111111",
+  TMPDIR => 1
```

Comment:
Looks like the temp files are not cleaned up after the testing is done, so I think we should be explicit about that by adding `UNLINK => 1` to the options.

Reply:
I added `UNLINK => 1,` to make sure the temporary file is removed upon exit. I also removed `TMPDIR => 1`, and `"XXXXX"` as the defaults seem to be enough (on Linux; awaiting test results from more platforms).